Code/Tools/ProjectManager/Source/LinkWidget.cpp

```diff
@@ -26,7 +31,29 @@ namespace O3DE::ProjectManager
     {
         if (m_url.isValid())
         {
-            QDesktopServices::openUrl(m_url);
+            // Check if user request not to be shown external link warning dialog
+            bool skipDialog = false;
+            QString settingsKey = GetExternalLinkWarningKey();
+            auto settingsRegistry = AZ::SettingsRegistry::Get();
+
+            if (settingsRegistry)
+            {
+                settingsRegistry->Get(skipDialog, settingsKey.toStdString().c_str());
+            }
```

Comment:
Could move the call to `GetExternalLinkWarningKey` inside the if statement since it is only used there.  Similarly, `settingsRegistry` could be initialized as part of the if statement instead.